### PR TITLE
Updating chart tags for cis-operator and security-scan images and set version to v0.2.0

### PR DESCRIPTION
**Pull request by prachidamle**
Updating chart tags for cis-operator and security-scan images and setting app version to v0.2.0

https://github.com/rancher/cis-operator/issues/37

### CODE REVIEW

File: packages/rancher-cis-benchmark/charts/values.yaml [hunk 1]
```diff
@@ -5,10 +5,10 @@
 image:
   cisoperator:
     repository: rancher/cis-operator
-    tag: v0.0.6
+    tag: v0.0.7
   securityScan:
     repository: rancher/security-scan
-    tag: v0.2.0
+    tag: v0.2.1
```
Comment (cbron) [inline packages/rancher-cis-benchmark/charts/values.yaml:11]: are these images already pushed/mirrored in dockerhub ?

File: packages/rancher-cis-benchmark/charts/Chart.yaml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 apiVersion: v1
-appVersion: v0.0.1
+appVersion: v0.2.0
 description: The cis-operator enables running CIS benchmark security scans on a kubernetes
   cluster
 name: rancher-cis-benchmark
-version: 0.0.1
+version: 0.2.0
```
Comment (cbron) [inline packages/rancher-cis-benchmark/charts/Chart.yaml:6]: Are we removing old version of chart ?

Comment (prachidamle) [reply packages/rancher-cis-benchmark/charts/Chart.yaml:6]: yes we want to start the app version start from v0.2.0 - I hope we need to change both version and appVersion fields?

File: packages/rancher-cis-benchmark/charts/values.yaml (hunk 1 above)
Comment (prachidamle) [reply packages/rancher-cis-benchmark/charts/values.yaml:11]: yes they have been tagged and built- here https://github.com/rancher/security-scan/releases/tag/v0.2.1

File: packages/rancher-cis-benchmark/charts/Chart.yaml (hunk 2 above)
Comment (cbron) [reply packages/rancher-cis-benchmark/charts/Chart.yaml:6]: The version corresponds to the version of the chart, the appVersion is the version of the operator. Thats why we often use the double-zero method and keep them in line. IE: operator version 1.2.1 gets `appVersion` of 1.2.1 and `version` of 1.2.100.

If we are removing the old version of the chart, you'll need to remove from the dev-v2.5 branch the entry in `assets/index.yaml`, and the assets.

**Review by aiyengar2 — DISMISSED** [review]

File: packages/rancher-cis-benchmark/charts/Chart.yaml (hunk 2 above)
Comment (prachidamle) [reply packages/rancher-cis-benchmark/charts/Chart.yaml]: Ok, Added this new PR for assets changes for dev-v2.5

https://github.com/rancher/charts/pull/716

File: packages/rancher-cis-benchmark/charts/Chart.yaml [hunk 3]
```diff
@@ -1,9 +1,9 @@
 apiVersion: v1
-appVersion: v0.0.1
+appVersion: v0.2.0
 description: The cis-operator enables running CIS benchmark security scans on a kubernetes
   cluster
 name: rancher-cis-benchmark
-version: 0.0.1
+version: 0.2.000
```
Comment (cbron) [inline packages/rancher-cis-benchmark/charts/Chart.yaml:6]: Swapping this for 1.0.000 now

Comment (prachidamle) [reply packages/rancher-cis-benchmark/charts/Chart.yaml]: Swapped to 1.0.000  and also added a logo link and the logo image is added to dev-2.5 PR - is that correct @cbron

File: packages/rancher-cis-benchmark/charts/Chart.yaml [hunk 4]
```diff
@@ -1,9 +1,10 @@
 apiVersion: v1
-appVersion: v0.0.1
+appVersion: v1.0.0
 description: The cis-operator enables running CIS benchmark security scans on a kubernetes
   cluster
 name: rancher-cis-benchmark
-version: 0.0.1
+version: 1.0.000
+icon: https://charts.rancher.io/assets/logos/cis-kube-bench.svg
```
Comment (cbron) [inline packages/rancher-cis-benchmark/charts/Chart.yaml:7]: This doesn't actually exist

**Review by cbron — APPROVED** [review]
looks good beside missing logo

File: packages/rancher-cis-benchmark/charts/Chart.yaml (hunk 4 above)
Comment (prachidamle) [reply packages/rancher-cis-benchmark/charts/Chart.yaml:7]: Seems like all the logo images are in dev-2.5 branch - so I added it to dev-2.5 PR

Comment (prachidamle) [reply packages/rancher-cis-benchmark/charts/Chart.yaml:7]: Is this the way we need to follow? or there is some other method to get this added? @cbron

Comment (cbron) [reply packages/rancher-cis-benchmark/charts/Chart.yaml:7]: Whoops, missed your message above. Yea that works, so this PR is good.

**Pull request merged**